packages/salesforcedx-vscode-core/src/commands/functions/functionService.ts

```diff
@@ -232,9 +225,7 @@ export class FunctionService {
       port: debugPort
     };
 
-    if (functionExecution.isContainerLess) {
-      delete debugConfiguration.remoteRoot;
-    }
+    delete debugConfiguration.remoteRoot;
```

Comment:
can we just remove the removeRoot property from the `getDebugConfiguration` function?

Reply:
we'll need to remove it from [here](https://github.com/forcedotcom/salesforcedx-vscode/blob/b5bacd0dd44e6d8b4e2e1f6a593eb8af64f40181/packages/salesforcedx-vscode-core/test/vscode-integration/commands/functions/functionService.test.ts#L106) as well